src/ir.cpp

```diff
@@ -757,7 +757,9 @@ Stmt CallStmt::make(std::vector<Var> results,
 
 // struct Scope
 Stmt Scope::make(Stmt scopedStmt) {
-  iassert(scopedStmt.defined());
+  if (!scopedStmt.defined()) {
```

Comment:
Is this really necessary? Looks like it just papers over mistaken logic in building IR? Do leave it in if it's really convenient though!